index.html

```diff
@@ -10,7 +10,145 @@
+                fill="#6070FF" />
+        </svg>
```

Comment:
- [ ] Kindly make the logo of the page a link by using the `a` tag, this way when the user clicks on it, it takes him to the top of the page 👍 

[Optional]

- [ ] I think it's better if you just write it as a text, like this `<a href="#">My Logo<a>`. It's shorter and less messy than the `svg`